templates/launch.html

```diff
@@ -140,19 +173,81 @@ <h3>${survey_type} Survey Metadata</h3>
         `
     }
 
-    function loadMetadata() {
+    function clearAndDisableSchemaUrlSelection() {
+        let schemaUrlElement = document.querySelector("#schema-url")
+        let surveyTypeElement = document.querySelector("#schema-url-survey-type")
+
+        surveyTypeElement.selectedIndex = 0
+        schemaUrlElement.value = ""
+        surveyTypeElement.disabled = true
+        schemaUrlElement.disabled = true
```

Comment:
Can we disable the "Load schema" button here as well?

Reply:
Was being done but in the wrong place, have updated.